packages/component-base/src/element-mixin.js

```diff
@@ -48,9 +48,10 @@ export const ElementMixin = (superClass) =>
         window.Vaadin.registrations.push(this);
         registered.add(is);
 
-        if (window.Vaadin.developmentModeCallback) {
+        const callback = window.Vaadin.developmentModeCallback;
+        if (callback) {
```

Comment:
Without this change, I was getting the following error in the `dist`:

```
element-mixin-brp7V6lp.js:16 Uncaught TypeError: Cannot read properties of undefined (reading 'vaadin-usage-statistics')
    at f._callback (element-mixin-brp7V6lp.js:16:525)
    at element-mixin-brp7V6lp.js:11:1075
```

Storing the callback into variable fixes it.